junifer/preprocess/tests/test_bold_warper.py

```diff
@@ -106,3 +106,75 @@ def test_BOLDWarper_preprocess(
         )
         assert data_type == "BOLD"
         assert isinstance(data, dict)
+
+
+@pytest.mark.parametrize(
+    "datagrabber, element, space",
+    [
+        [
+            DMCC13Benchmark(
+                types=["BOLD"],
+                sessions=["wave1bas"],
+                tasks=["Rest"],
+                phase_encodings=["AP"],
+                runs=["1"],
+                native_t1w=False,
+            ),
+            ("f9057kp", "wave1bas", "Rest", "AP", "1"),
+            "MNI152NLin2009aAsym",
+        ],
+        [
+            DMCC13Benchmark(
+                types=["BOLD"],
+                sessions=["wave1bas"],
+                tasks=["Rest"],
+                phase_encodings=["AP"],
+                runs=["1"],
+                native_t1w=False,
+            ),
+            ("f9057kp", "wave1bas", "Rest", "AP", "1"),
+            "MNI152NLin6Asym",
+        ],
+        [
+            DMCC13Benchmark(
+                types=["BOLD"],
+                sessions=["wave1bas"],
+                tasks=["Rest"],
+                phase_encodings=["AP"],
+                runs=["1"],
+                native_t1w=False,
+            ),
+            ("f9057kp", "wave1bas", "Rest", "AP", "1"),
+            "MNI152NLin2009cAsym",
+        ],
+    ],
+)
+@pytest.mark.skipif(
+    _check_ants() is False, reason="requires ANTs to be in PATH"
+)
+def test_BOLDWarper_preprocess_to_multi_mni(
+    datagrabber: "BaseDataGrabber", element: Tuple[str, ...], space: str
+) -> None:
+    """Test BOLDWarper preprocess.
+
+    Parameters
+    ----------
+    datagrabber : DataGrabber-like object
+        The parametrized DataGrabber objects.
+    element : tuple of str
+        The parametrized elements.
+    space : str
+        The parametrized template space to transform to.
+
+    """
+    with datagrabber as dg:
+        # Read data
+        element_data = DefaultDataReader().fit_transform(dg[element])
```

Comment:
Can we add a check to see if the warped images are different from the original? It's very basic, but is something.

Reply:
Addressed in the latest commit. Tests pass in juseless.